### PR TITLE
SQL: Period Interpolation tests

### PR DESCRIPTION
**Pull request by unrealwork**
## SQL: Period Interpolation tests (#1475)

*Contains tests that check different interpolation options*

## Review tasks
- [x] Rename `SqlExecuteMethod` to  `SqlMethod`
- [x]  Rename `addTestSamplesToSeries` to `sendSamplesToSeries`
- [x] Add tickets' number to tests
- [x] Rename `initialize` to `prepareDataSet`

### CODE REVIEW

**Comment by rmakulov** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/method/sql/period/interpolation/SqlPeriodInterpolationTests.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    }
+
+
+    /**
+     * Following tests related to #1475 task
+     * SQL: period() interpolation
+     */
+    @Test
+    public void testNoInterpolation() {
+        final String sqlQuery = "SELECT entity, datetime, AVG(value)\n" +
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/period/interpolation/SqlPeriodInterpolationTests.java]: what the `\n`?

Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/period/interpolation/SqlPeriodInterpolationTests.java]: For readability of SQL queries in logs

**Comment by Sild** [comment]
ок

File: src/test/java/com/axibase/tsd/api/method/sql/period/interpolation/SqlPeriodInterpolationTests.java [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * @author Igor Shmagrinskiy
+ */
+public class SqlPeriodInterpolationTests extends SqlExecuteMethod {
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/sql/period/interpolation/SqlPeriodInterpolationTests.java]: SqlExecuteMethod -> SqlMethod?

File: src/test/java/com/axibase/tsd/api/method/sql/period/interpolation/SqlPeriodInterpolationTests.java [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    private static final String TEST_PREFIX = "sql-period-interpolation";
+    private static final Set<String> DEFAULT_ROW_FILTER = new HashSet<>(Arrays.asList("datetime", "AVG(value)"));
+
+
+    private static void addTestSamplesToSeries(Series series, Sample... samples) {
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/sql/period/interpolation/SqlPeriodInterpolationTests.java]: rename to sendSamplesToSeries(series, ...samples) (addTestSamplesToSeries seems add samples to series object but not sending to atsd)

File: src/test/java/com/axibase/tsd/api/method/sql/period/interpolation/SqlPeriodInterpolationTests.java [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+            throw new IllegalStateException("Failed to insert series: " + series);
+        }
+        if (!isSuccessInsert) {
+            throw new IllegalStateException("Failed to insert series: " + series);
+        }
+    }
+
+
+    @BeforeClass
+    public static void initialize() {
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/sql/period/interpolation/SqlPeriodInterpolationTests.java]: prepareDataset?

**Comment by KutuzovGregory** [comment]
Please annotate every test method with issue number

**Comment by KutuzovGregory** [comment]
ok

**Pull request merged**